Add guide on alternative migration method for thanos to mimir (#3554)
grafana/mimir: docs/sources/migration-guide/migrating-from-thanos-to-mimir-with-thanos-sidecar.md

---

> title: "Migrating from Thanos to Mimir using Thanos Sidecar"
> menuTitle: "Migrating from Thanos to Mimir using Thanos Sidecar"

Reviewer: ```suggestion
title: "Migrate from Thanos to Mimir using a Thanos sidecar"
menuTitle: "Migrate from Thanos to Mimir using a Thanos sidecar"
```

> description: "Learn how to migrate from Thanos to Mimir using the Thanos Sidecar"

Reviewer: ```suggestion
description: "Learn how to migrate from Thanos to Mimir using Thanos sidecar"
```

weight: 10
---

> # Migrating from Thanos to Mimir using Thanos Sidecar
> 
> This document guides an operator through the process of migrating a deployment of Thanos to Grafana Mimir making use of the Thanos sidecar to move their metrics over with a stepped workflow.

Reviewer: ```suggestion
# Migrate from Thanos to Mimir using Thanos sidecar

As an operator, you can migrate a deployment of Thanos to Grafana Mimir by using Thanos sidecar to move your metrics over with a stepped workflow.
```
